### PR TITLE
Karl/homepage content

### PR DESCRIPTION
**Pull request by Karl-J-Bruder**
Fixed inline styles and added map keys

### CODE REVIEW

File: src/components/molecules/ImageGallery/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components"
+import { space } from "../../../styles/vars"
+
+export const StyleImageGallery = styled.div`
+    border: 2px solid red;
+    display: flex;
+    flex-direction: row;
+    height: 300px;
+    margin: ${space.l} 0 ${space.l} 0;
```
Comment (Alfrex92) [inline src/components/molecules/ImageGallery/styles.tsx]: you can do this.
` margin: ${space.l} 0; `

File: src/components/molecules/ScrollingGrid/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { StyleScrollingGridWrapper, StyleGridUL, StyleGridLI } from "./styles"
+import HorizontalScrollChild from "../../atoms/GalleryImage"
+import data from "../../../../config/translations/en.json"
```
Comment (Alfrex92) [inline src/components/molecules/ScrollingGrid/index.tsx]: `data` should come from props.

Comment (Alfrex92) [reply src/components/molecules/ScrollingGrid/index.tsx]: remember that the component needs to be agnostic of what data will receive.

File: src/components/molecules/ScrollingGrid/styles.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components"
+
+export const StyleScrollingGridWrapper = styled.div`
+    display: grid;
+    grid-template-columns: 20px 1fr 20px;
+
+    & > * {
```
Comment (Alfrex92) [inline src/components/molecules/ScrollingGrid/styles.tsx]: I avoid using the `*`, well I haven't been in a situation where I need it. I'm not saying that is a bad idea, though. 
You might want to read this article. 
https://www.clairecodes.com/blog/2019-04-21-my-misconceptions-about-the-universal-selector-in-css/

File: src/components/molecules/ScrollingGrid/styles.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        background-color: transparent;
+    }
+    ::-webkit-scrollbar-thumb {
+        background-color: transparent;
+      }
+`;
+
+export const StyleGridUL = styled.ul`
+    display: grid;
+    grid-gap: 10px;
```
Comment (Alfrex92) [inline src/components/molecules/ScrollingGrid/styles.tsx]: use a variable.

File: src/components/molecules/ScrollingGrid/styles.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  grid-template-rows: minmax(150px, 1fr);
+    &::before,
+    &::after {
+        content: ‘’;
+    }
+`;
+
+export const StyleGridLI = styled.li`
+      scroll-snap-align: center;
+  padding: calc(var(--gutter) / 2 * 1.5);
```
Comment (Alfrex92) [inline src/components/molecules/ScrollingGrid/styles.tsx]: We're not using CSS variables in this project. Why suddenly use a CSS Variable? If you want to use a variable create the variable inside the file `styles/vars` 
Note: I'm not against using CSS variables, but I think that this is the first time that we use it. Let's keep the consistency.
Note 2: For this project, we don't support old browsers like IE but be aware that CSS variables are not supported in old browsers.

File: src/components/molecules/ScrollingGrid/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
```
Comment (Alfrex92) [inline src/components/molecules/ScrollingGrid/styles.tsx]: add a variable here.

**Review by Alfrex92 — COMMENTED** [review]
You can merge once you apply the changes 😄

File: src/components/molecules/ScrollingGrid/styles.tsx (hunk 5 above)
Comment (Karl-J-Bruder) [reply src/components/molecules/ScrollingGrid/styles.tsx]: I was basing this off of an example I found (you might have showed me the page): https://uxdesign.cc/creating-horizontal-scrolling-containers-the-right-way-css-grid-c256f64fc585

But I don't think this needs to be used anywhere else, so I'll just use an existing space variable instead.

**Pull request merged**